classes/VoiceTracker.js

```diff
@@ -25,29 +25,40 @@ class VoiceTracker {
         PRIMARY KEY (ServerId, UserId)
       )
     `)
+    try {
+      this.db.exec(`SELECT Active FROM VoiceStats`)
+    }
+    catch (error) {
+      this.db.exec(`ALTER TABLE VoiceStats ADD COLUMN Active INTEGER DEFAULT 0`)
+    }
   }
 
   initClient () {
     this.client.once("ready", () => {
-      setInterval(() => {
-        const d = new Date()
-        if (d.getHours() === 19 && d.getMinutes() === 0) {
-          const servers = Object.keys(this.db.get("servers").value())
+      this.client.guilds.cache.forEach(serverId => {
+        serverId.members.cache.forEach(mem => {
+          this.voiceStateUpdate(mem, mem)
+        })
+      })
+      // setInterval(() => {
```

Comment:
@MikeDev96 can we please make sure we don't have commented code in the production release?